lib/obsolete.gd

```diff
@@ -55,6 +55,17 @@
 ##  Usually, old names will be available for several releases;
 ##  they may be removed when they don't seem to be used any more.
 ##  <P/>
+##  Information about obsolete names is printed by <Ref Func="Info"/> using the
+##  InfoObsolete InfoClass. By default InfoObsolete is set to 1. Newly
+##  obsoleted identifiers should be first be outputted at info level 2. Once they
```

Comment:
"be first be outputted" ? Perhaps "at first be output" ("output" is its own past tense, like "read"). Though perhaps "at first be set to output at info level 2" is even better?